src/modules/blocks/ticket/container-header/quantity/style.pcss

```diff
@@ -1,26 +1,42 @@
 .tribe-editor__ticket__container-header-quantity {
-	margin-right: 13px;
 	min-width: 105px;
 }
 
 .tribe-editor__ticket__container-header-quantity-label {
+	color: var(--tec-color-text-primary);
 	display: block;
-	color: #000;
-	font-size: 15px;
-	line-height: 18px;
-	margin-bottom: 12px;
-	text-align: center;
+	font-family: var(--tec-font-family-sans-serif);
+	font-size: var(--tec-font-size-3);
+	font-style: normal;
+	font-weight: var(--tec-font-weight-bold);
+	line-height: 26px;
+	text-align: right;
 
 	.tribe-editor__ticket--disabled & {
 		color: #AEB4BB;
 	}
+
+	& > .dashicons-info-outline {
+		color: #727272;
+		font-family: dashicons;
+		font-feature-settings: 'clig' off, 'liga' off;
+		font-size: var(--tec-font-size-3);
+		font-style: normal;
+		font-weight: var(--tec-font-weight-regular);
+		line-height: 26px;
+		margin-left: 6px;
```

Comment:
Spacer var. In "close but not quite" cases like this I would reach out to design, etc and see if 4 (--tec-spacer-0) or 8 (--tec-spacer-1) are acceptable.

Reply:
Using tec spacers now